### PR TITLE
Add ViewTreeObserver GlobalLayout event binding

### PR DESCRIPTION
**Pull request by xfumihiro**
Refs #87

### CODE REVIEW

File: rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEventOnSubscribe.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    this.view = view;
+  }
+
+  @Override public void call(final Subscriber<? super ViewTreeObserverGlobalLayoutEvent> subscriber) {
+    checkUiThread();
+
+    final ViewTreeObserver.OnGlobalLayoutListener listener = new ViewTreeObserver.OnGlobalLayoutListener() {
+      @Override public void onGlobalLayout() {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN) {
+          view.getViewTreeObserver().removeOnGlobalLayoutListener(this);
```
Comment (dlew) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEventOnSubscribe.java]: Won't this mean there's only one emission before it stops?

Comment (xfumihiro) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEventOnSubscribe.java]: It's a one-shot triggering for typical use cases (e.g., getting the measured width & height).

Comment (xfumihiro) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEventOnSubscribe.java]: It should be handled "Reactively". I'll remove it.

File: rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java [hunk 2]
```diff
@@ -181,6 +182,20 @@
...
 
   /**
+   +   * Create an observable of {@link ViewTreeObserverGlobalLayoutEvent} for draws on {@code view}.
+   +   * <p>
+   +   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   +   * to free this reference.
+   +   * <p>
+   +   * <em>Warning:</em> The created observable uses {@link ViewTreeObserver#addOnGlobalLayoutListener} to observe
+   +   * globalLayouts. Multiple observables can be used for a view at a time.
+   +   */
```
Comment (JakeWharton) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java]: Screwed up Javadoc here. Remove all the `+`'s and fix indent.

File: rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEventOnSubscribe.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  public ViewTreeObserverGlobalLayoutEventOnSubscribe(View view) {
+    this.view = view;
+  }
+
+  @Override public void call(final Subscriber<? super ViewTreeObserverGlobalLayoutEvent> subscriber) {
+    checkUiThread();
+
+    final ViewTreeObserver.OnGlobalLayoutListener listener = new ViewTreeObserver.OnGlobalLayoutListener() {
+      @Override public void onGlobalLayout() {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN) {
```
Comment (JakeWharton) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEventOnSubscribe.java]: The `if` can be removed. Both branches are the same.

File: rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEventOnSubscribe.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        }
+
+      }
+    };
+
+    view.getViewTreeObserver().addOnGlobalLayoutListener(listener);
+
+    subscriber.add(new MainThreadSubscription() {
+      @Override protected void onUnsubscribe() {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN) {
```
Comment (JakeWharton) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEventOnSubscribe.java]: This should be `>=`

File: rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEvent.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
+package com.jakewharton.rxbinding.view;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.view.View;
+
+public final class ViewTreeObserverGlobalLayoutEvent extends ViewEvent<View> {
```
Comment (JakeWharton) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEvent.java]: I'm wondering whether it's useful to actually have an event object for this? Or should we just emit an `Object` instance as a signal (like with clicks)?

File: rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java [hunk 6]
```diff
@@ -181,6 +182,35 @@
...
+   * Create an observable of {@link ViewTreeObserverGlobalLayoutEvent} for globalLayouts on {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses {@link ViewTreeObserver#addOnGlobalLayoutListener} to observe
+   * globalLayouts. Multiple observables can be used for a view at a time.
+   */
+  @CheckResult @NonNull
+  public static Observable<ViewTreeObserverGlobalLayoutEvent> globalLayoutEvents(@NonNull View view) {
```
Comment (JakeWharton) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java]: Let's delete this per https://github.com/JakeWharton/RxBinding/pull/121/files#r40500931

**Pull request merged**